ci/requirements/py36-min-all-deps.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - cartopy=0.17
   - cdms2=3.1
   - cfgrib=0.9
-  - cftime=1.0
+  - cftime=1.0.0
```

Comment:
@spencerkclark do we want to bump this up to 1.0.3?

Reply:
Thanks for doing this @dcherian.  Yes, I think that is appropriate, given the [new minimum dependency rules](http://xarray.pydata.org/en/stable/installing.html#minimum-dependency-versions); version 1.0.3 was released about 10 months ago.  

It would be nice at some point to remove some of the cruft that is leftover from supporting both `netcdftime` and `cftime` in places.  Maybe when I get a free moment I'll look into that.

Reply:
:+1: done.